### ISSUE TITLE
Consider renaming to Chromium

### ISSUE THREAD

**Issue by RByers** [ArthurSonzogni/ChromeCommitTracker#10]
I hate to nitpick, but as this tracker is increasingly used by the Chromium contributor community, I'm feeling more and more that the name "Chrome Commit Tracker" is not ideal. Eg. I think this is what we should now be using in our BlinkOn keynote slides, but really we'd want to emphasize "Chromium" there, not "Chrome". I don't think it's worth a ton of work, but if you ever feel so inclined it would be awesome if you someday renamed it to "Chromium Commit Tracker".

**Comment by RByers** [comment]
Maybe updating the UI strings without changing the domain name or repo name would be the right pragmatic cost/benefit tradeoff?

**Comment by ArthurSonzogni** [comment]
> Maybe updating the UI strings without changing the domain name or repo name would be the right pragmatic cost/benefit tradeoff?

I agree, updating the UI strings without changing the domain or repo name seems like the most pragmatic approach right now. It minimizes disruption while addressing this issue.

I believe Github supports renaming repositories gracefully, providing the right redirects. However updating the domain name will require some careful considerations to avoid breaking previously issued links. I will do it eventually.

About the title. Here is a concept to make it clearer this is about tracking any repositories related to chromium::
![test](https://github.com/user-attachments/assets/1b946c30-efc2-4ffe-af18-c837a42d463e)

**Comment by ArthurSonzogni** [comment]
Deployed soon: https://github.com/ArthurSonzogni/ChromeCommitTracker/actions/runs/10887865288/job/30211043795

**Comment by RByers** [comment]
Nice, cool! Yeah you could just call it ArthurSozongni/CommitTracker if you like 😊. I've renamed GitHub repos before for standards work and found it fairly painless (but with some edge cases). IIRC GitHub does URL redirects pretty seamlessly.
